site/en/docs/webstore/faq/index.md

```diff
@@ -174,7 +175,7 @@ The item won't appear in the web store until it has successfully completed the r
 details, see [How long will it take to review my item?][how-long-do-reviews-take].
 
 Extensions which are published to the same domain as the publisher address may be approved more
-quickly.
+quickly. Learn more about [Enterprise publishing][enterprise].
```

Comment:
Above you capitalize Enterprise, here you don't. I suspect lowercase is better.

Reply:
Good catch!